cli/test/lib/cypress_spec.js

```diff
@@ -172,4 +172,30 @@ describe('cypress', function () {
       })
     })
   })
+
+  context('cli', function () {
+    describe('.parseRunArguments', function () {
+      it('parses CLI cypress run arguments', async () => {
+        const args = 'cypress run --browser chrome --spec "my/test/spec.js"'.split(' ')
+        const options = await cypress.cli.parseRunArguments(args)
+
+        // TODO remove quotes around specs and other string arguments
+        expect(options).to.deep.equal({
+          browser: 'chrome',
+          spec: '"my/test/spec.js"',
```

Comment:
Should we be echo-ing the exact values back or the resolved spec paths?

Reply:
well, the double quotes are probably NOT what the user expects to see since they are for the shell to escape things like spaces in the file paths. I would remove it before returning the object to the user.

Reply:
scratch this, the shell would not pass `"` into the CLI, so this particular case is an artifact that we call this as a unit test